bin/run_robot

```diff
@@ -3,7 +3,7 @@
 #
```

Comment:
you can copy this scriptie from common-accessioning -- https://github.com/sul-dlss/common-accessioning/blob/master/bin/run_robot, or doesn't that one work?